.github/workflows/test.yml

```diff
@@ -2,9 +2,9 @@ name: Tests
 
 on:
   push:
-    branches: [ master, rc ]
+    branches: [ master, rc, v4 ]
   pull_request:
-    branches: [ master, rc ]
```

Comment:
Except for `test-e2e`, maybe - any reason not to run on all branches? Just compute time / actions-minutes?

Reply:
I was more tacking on to existing stuff, but yeah, I think this makes sense for now. At the very least, my Program API PRs target a feature branch where we know the tests aren't going to pass, so it saves us a good amount of compute time.